config-linux.md

```diff
@@ -549,8 +549,14 @@ The following parameters can be specified for the container:
 
     * If `closID` is set, and neither of `l3CacheSchema` and `memBwSchema` are set, runtime MUST check if corresponding pre-configured directory `closID` is present in mounted `resctrl`. If such pre-configured directory `closID` exists, runtime MUST assign container to this `closID` and [generate an error](runtime.md#errors) if directory does not exist.
 
-* **`monitoring`** *(boolean OPTIONAL)* - specifies if Intel RDT monitoring features (CMT and MBM) should be enabled
-
+* **`monitoring`** *(boolean OPTIONAL)* - specifies if Intel RDT monitoring features:
+    * CMT ( Cache Monitoring Technology ) supports monitoring of the last-level cache (LLC) occupancy
+      for each running thread simultaneously.
+    * MBM ( Memory Bandwidth Monitoring ) supports monitoring of total and local memory bandwidth
+      for each running thread simultaneously.
+    
+    should be enabled.
```

Comment:
Could this line be moved to line 552?
* **`monitoring`** *(boolean OPTIONAL)* - specifies if Intel RDT monitoring features as below should be enabled:

Reply:
Done